packages/api/src/graphql/comment/comment.private-mutation.ts

```
@@ -46,12 +52,35 @@ export const updateComment = async (
   guestUserImageID: string,
   source: string,
   tagIds: string[] | undefined,
+  ratingOverrides: CommentRatingOverrideInput[] | undefined,
   authenticate: Context['authenticate'],
+  commentRatingAnswerClient: PrismaClient['commentRatingSystemAnswer'],
   commentClient: PrismaClient['comment']
 ) => {
   const {roles} = authenticate()
   authorise(CanUpdateComments, roles)
 
+  if (ratingOverrides?.length) {
+    const answerIds = ratingOverrides.map(override => override.answerId)
+    const answers = await commentRatingAnswerClient.findMany({
+      where: {
+        id: {
+          in: answerIds
+        }
+      }
+    })
+
+    ratingOverrides.forEach(override => {
+      const answer = answers.find(a => a.id === override.answerId)
+
+      if (!answer) {
+        return
```

should not happen. Therefore, we should throw an error.